libs/methodologies/bold/rule-processors/mass-id/avoided-emissions/src/README.md

```diff
@@ -0,0 +1,17 @@
+# BOLD Carbon CH₄ – Baseline Emission Factors
+
+This module provides precomputed baseline methane emissions for organic waste disposal scenarios as defined in the **BOLD Carbon CH₄ Methodology**.
+
+## 📘 Context
+
+When calculating avoided emissions for composting projects, it is essential to compare each material's potential emissions under traditional waste disposal scenarios. These reference scenarios (baselines) are:
+
+- **Landfills with methane flaring**
+- **Landfills without methane flaring**
+- **Open-air dumps**
+
+Each organic waste category is assigned emission factors (in **tons of CO₂e per ton of waste**) based on default values and decay models aligned with **AMS-III.F v12.0** and IPCC standards.
```

Comment:
Just to make sure, is *decay* the right word in this context? Is this based on some reference text?

Reply:
It makes sense to me!